Storage/SQL/BrowserDB.swift

```diff
@@ -342,13 +343,15 @@ extension BrowserDB {
      * the callers thread until they've finished. If any of them fail the operation will abort (no more
      * commands will be run) and the transaction will rollback, returning a DatabaseError.
      */
-    func run(sql: [(sql: String, args: Args?)]) -> Success {
+    func run(sql: [(sql: String?, args: Args?)]) -> Success {
```

Comment:
I'm not at all keen on this change; it still calls into a transaction and creates a `Deferred`, and it seems like it'll hide errors.

I'd rather have the subset of callers that might not end up with a query do the check early.